src/main/java/com/github/vkorobkov/jfixturescmd/CmdArgs.java

```diff
@@ -0,0 +1,33 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.beust.jcommander.Parameter;
+import lombok.Getter;
+
+@Getter
+final class CmdArgs {
+    @Parameter(names = "-type", description = "SQL type", required = true,
+            converter = SqlTypesConverter.class, order = 0)
+    private SqlTypes sqlType;
+
+    @Parameter(names = "-src", description = "Source YAML folder path", required = true, order = 1)
+    private String source;
+
+    @Parameter(names = "-dst", description = "Destination SQL file path", required = true, order = 2)
+    private String destination;
+
+    @Parameter(names = { "-h", "--help" }, help = true)
+    private boolean help;
+
+    private CmdArgs() {
+    }
+
+    private static final class CmdArgsHelper {
```

Comment:
Just a though - maybe just 
```java
public final static CmdArgs instance = new CmdArgs()
```
?

Reasons:
1. We don't have a need for lazy loading here(class gets instantiated every time the program launches)
2. Class don't have any logic in constructor and we don't have to handle any specific errors when call `getInstance()`